<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaybackPage</class>
 <widget class="QMainWindow" name="PlaybackPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="lastRound">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>480</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Round</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextRound">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>480</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Round</string>
    </property>
   </widget>
   <widget class="QPushButton" name="autoPlay">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>480</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Autoplay</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
